src/clr-angular/button/button-group/_button-group.clarity.scss

```diff
@@ -15,6 +15,27 @@
     .btn {
       margin: 0;
       vertical-align: top;
+      overflow: visible;
+
+      label {
+        height: 100%;
+      }
+
+      input[type='radio']:focus,
+      input[type='checkbox']:focus {
+        appearance: none;
+        opacity: 1;
+        top: 0;
+        height: 100%;
+        width: 100%;
+        box-shadow: 0 0 5px rgba(81, 203, 238, 1);
```

Comment:
Don't we need `outline:none` as we are providing our own custom focus outline style here? 
Also, the color used seems to be very specific. Do we care whether this color is accessible or not?

Reply:
That's a great question. I'm a bit torn on this, because the native outline provides the familiar visual indicator for focused state in chrome, safari, and IE so it's more recognizable even though we now have the added box shadow on top of that. In IE11, the box shadow isn't that prominent without the native outline (dotted line). If I change the box shadow property to make it more noticeable in IE11, chrome and safari look very different from the native outline style. I guess I'm not opposed to that but it'd seem like a bigger visual change and departure from the native outline.

I copied the actual color used here from a gist, and the idea is to mimic the glow color with it. I'm open to it being a different color (cc @lil-kim, @colinreedmiller) but we should keep it close to the focus color that's used throughout our components. 

I believe the color contrast ratio requirement for a11y is for distinguishing text against a background color.. For example, safari's focus color is super faint by default and what you see in the screenshot is that.

Reply:
What do we need the explicit border for?
I tried changing it to red and this was the result:
<img width="140" alt="screen shot 2018-12-11 at 11 41 14 am" src="https://user-images.githubusercontent.com/1798828/49791745-c2534380-fd39-11e8-9dec-711c366bd5aa.png">
(chrome on mac)

Reply:
I'm with Jeeyun on this one, keeping the browser's default focus (when it exists) on top of ours is better for people who rely on it. From the various screenshots posted, I don't think the double outline looks bad in any case.

Reply:
+1

Reply:
I'm also with Jeeyun on this one. The question was only for my information. Sorry if it sounded like a request for change. Thanks for the details :D

Reply:
@Shijir I think it's ok for now. It's confined to one section of our scss file and not really re-usable anywhere else. Might be worth it in future if we decide to have custom focus across all our components.

Reply:
Sounds good 👍